app/views/changesets/timeout.atom.builder

```diff
@@ -1,6 +1,6 @@
 atom_feed(:language => I18n.locale, :schema_date => 2009,
-          :id => url_for(params.merge(:only_path => false)),
-          :root_url => url_for(params.merge(:only_path => false, :format => nil)),
+          :id => url_for(params.permit(:format, :display_name, :bbox, :friends, :nearby).merge(:only_path => false)),
+          :root_url => url_for(params.permit(:format, :display_name, :bbox, :friends, :nearby).merge(:action => :index, :format => nil, :only_path => false)),
```

Comment:
We should probably factor out the `params.permit` into a variable to avoid duplicating it?

Reply:
Added a variable.

I didn't try to write clean `params.permit` code because some dubious code already exist. I was going to fix it later.